src/app/qgswelcomepageitemsmodel.cpp

```diff
@@ -44,9 +110,8 @@ QVariant QgsWelcomePageItemsModel::data( const QModelIndex& index, int role ) co
   switch ( role )
   {
     case Qt::DisplayRole:
-      return mRecentProjects.at( index.row() ).title;
+      return QString( "<span style='font-size:18px;font-weight:bold;'>%1</span><br>%2" ).arg( mRecentProjects.at( index.row() ).title != mRecentProjects.at( index.row() ).path ? mRecentProjects.at( index.row() ).title : QString( tr( "- untitled -" ) ) ).arg( mRecentProjects.at( index.row() ).path );
```

Comment:
In terms of a clean MVC approach, the style should probably completely go to the delegate and the model should provide raw data.